clients/CMakeLists.txt

```diff
@@ -1,5 +1,22 @@
 # ########################################################################
-# Copyright 2016-2022 Advanced Micro Devices, Inc.
+# Copyright (C) 2016-2022 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
nitpicking as no one wants to this again. I am not sure about spacing surrounding (C). I think you are right but should we confirm?

Reply:
I asked in the ticket but haven't heard back, I'll go ahead with this merge.